x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import { IRouter } from 'src/core/server';
+import { EndpointAppContext } from '../../types';
+
+/**
+ * Registers the Host-(un-)isolation routes
+ */
+export function registerHostIsolationRoutes(router: IRouter, endpointContext: EndpointAppContext) {
```

Comment:
is this from your PR? or from Dan's? 
if from yours make sure you add tests for both the routes and the service that supports these routes. I would also recomment spliting out the schema and adding test cases to those (unless you will validate them via tests for the handlers (which I am not sure how to do)).

Reply:
it's from dan's which will be fleshed out and completed in a different pr i believe